georel/package.json

```diff
@@ -22,6 +24,9 @@
     },
     "cds": {
         "requires": {
+            "auth": {
+                "kind": "dummy-auth"
```

Comment:
I am not quite understanding how does the "dummy-auth" work here? Is it provided due to specific error or due to cds version upgrade?

Reply:
Following issue is intended for this change.

> HI @TieyanFu ,
Please check this ticket https://github.tools.sap/cap/issues/issues/11445
Regards,
Lalith. S